set.mm

```diff
@@ -645574,6 +645741,54 @@ with respect to a linear (1-dimensional) relation.  (Contributed by BJ
       SUBUPLVCUTVDVEVFVGVHVIVJVTISIVTLVKVLDWJCKVNVOVJ $.
   $}
 
+  ${
+    $d W l x y $.  $d B x y $.  $d S l $.  $d .x. l x y $.  $d ph x y $.
+    prjspner.e $e |- .~ = { <. x , y >. |
+      ( ( x e. B /\ y e. B ) /\ E. l e. S x = ( l .x. y ) ) } $.
+    prjspner.w $e |- W = ( K freeLMod ( 0 ... N ) ) $.
+    prjspner.b $e |- B = ( ( Base ` W ) \ { ( 0g ` W ) } ) $.
+    prjspner.s $e |- S = ( Base ` K ) $.
+    prjspner.x $e |- .x. = ( .s ` W ) $.
+    prjspner.k $e |- ( ph -> K e. DivRing ) $.
+    $( The relation in ` PrjSpn ` is an equivalence relation.  (Contributed by
```

Comment:
The binary relation used to define `PrjSp` in ~ df-prjsp (hence also indirectly `PrjSpn` in ~ df-prjspn ) is an equivalence relation.

Reply:
Oh, I hadn't seen your already existing ~prjsper, which explains your wording. Anyway, the links you added help the reader connect the dots, so I think they're useful.